packages/playground/src/components/node_details_cards/cpu_benchmark_card.vue

```diff
@@ -50,12 +51,18 @@ export default {
     };
 
     onMounted(async () => {
+      if (!gridStore.grid) {
+        errorMessage.value = "Unable to load CPU Benchmark details; please connect your wallet and try again.";
+        return;
+      }
```

Comment:
Can we hide the cards and show errors only if grid is defined? This goes for Iperf too.